ts/components/SessionFocusTrap.tsx

```diff
@@ -0,0 +1,27 @@
+import FocusTrap from 'focus-trap-react';
+import React from 'react';
+import styled from 'styled-components';
+
+const DefaultFocusButton = styled.button`
+  opacity: 0;
+  width: 0;
+  height: 0;
+`;
+
+/**
+ * The FocusTrap always require at least one element to be tabbable.
+ * On some dialogs, we don't have any until the content is loaded, and depending on what is loaded, we might not have any tabbable elements in it.
+ * This component renders the children inside a FocusTrap and always adds an invisible button to make FocusTrap happy.
```

Comment:
I have come up with a slightly cleaner solution in my opinion. Have tested on my machine and confirmed working. Can you update your code and test please? We can also should test on windows to be 100% through.

```tx
import FocusTrap from 'focus-trap-react';
import React, { ReactNode, useState } from 'react';
import { useMount } from 'react-use';

type Props = {
  children: ReactNode;
};

export function SessionFocusTrap(props: Props) {
  const [active, setActive] = useState(false);

  useMount(() => setActive(true));

  return (
    <FocusTrap active={active} focusTrapOptions={{ initialFocus: false, allowOutsideClick: true }}>
      {props.children}
    </FocusTrap>
  );
}

```